enterprise/audit/table.go

```diff
@@ -133,6 +133,20 @@ var AuditableResources = auditMap(map[any]map[string]Action{
 		"quota_allowance": ActionTrack,
 		"members":         ActionTrack,
 	},
+	// We don't show any diff for the APIKey resource
+	&database.APIKey{}: {
```

Comment:
Another question @coadler: one of the next tickets I'm going to pick up is for [API token auditing](https://github.com/coder/coder/issues/4519). I think API Tokens use the same model as the API Key, right (`database.APIKey`)? How would you go about differentiating those two logs?

Reply:
Maybe we could have a second `APIToken` type that embeds `APIKey`? Or we could have a `type` field that specifies what the key is for.

Reply:
Under the hood aren't tokens and api keys the same?

Reply:
Yes, looks like it. What does 'embed `APIKey`' mean?

Reply:
```go
type APIToken struct {
    APIKey
}
```

I think I would probably prefer the `type` field method though.